DirectEmployers/MyJobs
# Upgrade js stuff 3

## What's this PR do?

Updates us to node 7 and replaces npm with yarn. Webpack dev server appears to work.

## Where should the reviewer start?

This is all Dockerfile changes.

## How should this be manually tested?

We should make sure all existing apps build and appear to work. We don't expect subtle breakage.

## Any background context you want to provide?

Run `dk rebuilddev`.

From now on we will not use `npm install`. Instead we will run `yarn`.
